<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTranslatorPlugin</class>
 <widget class="QWidget" name="FileTranslatorPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_fileTransTitle">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create a File Translator Plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbx_fileTransOptions">
       <property name="title">
        <string>File Translator Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="lbl_className">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Class Name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLineEdit" name="line_uiScriptName"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="lbl_transName">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>File Translator Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLineEdit" name="line_transName"/>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLineEdit" name="line_className"/>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="lbl_uiScriptName">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>UI Script Name</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLineEdit" name="line_fileExt"/>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QLabel" name="lbl_impExp">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Import/Export</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QLabel" name="lbl_defExt">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Default File Extension</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rdio_Import">
            <property name="text">
             <string>Import</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdio_export">
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rdio_impExp">
            <property name="text">
             <string>Import/Export</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QGroupBox" name="gbx_addPribAttribs">
          <property name="title">
           <string>Private Attributes</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="cbx_addPrivAttribs">
             <property name="text">
              <string>Add private class attributes</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_className</tabstop>
  <tabstop>line_transName</tabstop>
  <tabstop>line_uiScriptName</tabstop>
  <tabstop>rdio_Import</tabstop>
  <tabstop>rdio_export</tabstop>
  <tabstop>rdio_impExp</tabstop>
  <tabstop>line_fileExt</tabstop>
  <tabstop>cbx_addPrivAttribs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
